app/src/main/java/com/hedvig/app/util/extensions/LocalDateTimeExt.kt

```diff
@@ -5,8 +5,15 @@ import java.time.LocalDate
 import java.time.LocalDateTime
 import java.time.ZoneId
 
-fun Long.epochMillisToLocalDate(): LocalDateTime {
+fun Long.epochMillisToLocalDateTime(): LocalDateTime {
     return LocalDateTime.ofInstant(Instant.ofEpochMilli(this), ZoneId.systemDefault())
 }
 
+fun Long.epochMillisToLocalDate(): LocalDate {
+    return LocalDateTime.ofInstant(Instant.ofEpochMilli(this), ZoneId.systemDefault()).toLocalDate()
+}
+
+
 fun LocalDateTime.isToday() = toLocalDate().isEqual(LocalDate.now())
+
+fun LocalDate.isToday() = isEqual(LocalDate.now())
```

Comment:
From a readability-perspective, I quite this extension. However, it does something that might make testability harder: It hard-codes the `Clock` used to acquire `LocalDate::now`.

We might want to avoid that 😞

Reply:
Good point. Lets keep that in mind and change when needed!

Reply:
Good pint. Lets keep that in mind and change later when we need it.